quick_setup.py

```diff
@@ -241,12 +241,23 @@ async def _setup_postgres():
 
                 # Import and create all tables
                 from core.database.postgres_database import Base
-                from core.vector_store.pgvector_store import Base as VectorBase
 
                 # Create regular tables first
                 await conn.run_sync(Base.metadata.create_all)
                 LOGGER.info("Created base PostgreSQL tables")
 
+                # Create caches table
```

Comment:
Let's shift this to init.sql, that's a much cleaner way to set up and will reflect for docker as well.